service/src/test/java/bio/terra/cbas/controllers/TestMethodsApiController.java

```diff
@@ -44,6 +47,7 @@ class TestMethodsApiController {
   private static final String API = "/api/batch/v1/methods";
 
   @MockBean private CromwellService cromwellService;
+  @MockBean private DockstoreService dockstoreService;
```

Comment:
I'd also suggest adding `@ExtendWith(MockitoExtension.class)` to this test class

(you didn't change the class signature line, so this comment is just on the first changed line in the file)